src/main/scala/beam/utils/ParquetReader.scala

```diff
@@ -0,0 +1,70 @@
+package beam.utils
```

Comment:
There isn't a truly clean way to do it, so maybe this is the best option - but it might be worth documenting that you can also use [parquet-tools](https://github.com/apache/parquet-mr/tree/master/parquet-tools)

Reply:
Is not it command line tool? I'm not sure how I can use this that library just by itself. I can see an example of [CatCommand.java](https://github.com/apache/parquet-mr/blob/master/parquet-tools/src/main/java/org/apache/parquet/tools/command/CatCommand.java#L82) and how they read parquet file.

Reply:
Yah it is a command line tool - I thought that's what ParquetReader was too - I must have been mistaken